### PR TITLE
[MAINTENANCE] Use nested extension configuration

### PR DESCRIPTION
**Pull request by beatrycze-volk**
Over the time the configuration has grown quite significantly so introduction of nested configuration increases the readability.

### CODE REVIEW

**Comment by beatrycze-volk** [comment]
I would assume that it is not the error here:
```
Tests/Functional/FunctionalTestCase.php#L127
The use of function getenv() is discouraged
```

**Comment by beatrycze-volk** [comment]
Codacy warning about getevn() should be solved by #1206

File: ext_conf_template.txt [hunk 1]
```diff
@@ -1,102 +1,102 @@
...
+# cat=Basic; type=boolean; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.general.forceAbsoluteUrlHttps
+general.forceAbsoluteUrlHttps = 0
+# cat=Basic; type=boolean; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.general.caching
+general.caching = 0
+# cat=Basic; type=boolean; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.general.publishNewCollections
+general.publishNewCollections = 1
+# cat=Basic; type=boolean; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.general.unhideOnIndex
+general.unhideOnIndex = 0
+# cat=Basic; type=boolean; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.general.useExternalApisForMetadata
+general.useExternalApisForMetadata = 0
```
Comment (sebastian-meyer) [inline ext_conf_template.txt]:
```suggestion
# cat=General; type=string; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.general.userAgent
general.userAgent = Kitodo.Presentation
# cat=General; type=boolean; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.general.forceAbsoluteUrl
general.forceAbsoluteUrl = 0
# cat=General; type=boolean; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.general.forceAbsoluteUrlHttps
general.forceAbsoluteUrlHttps = 0
# cat=General; type=boolean; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.general.caching
general.caching = 0
# cat=General; type=boolean; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.general.publishNewCollections
general.publishNewCollections = 1
# cat=General; type=boolean; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.general.unhideOnIndex
general.unhideOnIndex = 0
# cat=General; type=boolean; label=LLL:EXT:dlf/Resources/Private/Language/locallang_labels.xlf:config.general.useExternalApisForMetadata
general.useExternalApisForMetadata = 0
```

**Review by sebastian-meyer — CHANGES_REQUESTED** [review]

**Review by sebastian-meyer — APPROVED** [review]

**Comment by fschoelzel** [comment]
Found missing nesting key for `metadataFormats` in `Resources/Private/Language/locallang_labels.xlf`
See pull Request: #1264

**Pull request merged**